algorithms/ar-msrt/javascript/merge_sort.js

```diff
@@ -0,0 +1,36 @@
+function merge(array1, array2) {
+    if (array1 === []) {
+        return array2;
+    }
+    else if (array2 === []) {
+        return array1;
+    }
+    else {
+        let output = [];
+        while (array1.length > 0 || array2.length > 0) {
+            if (array1.length === 0) {
+                output.push(array2.shift());
+            }
+            else if (array2.length === 0) {
+                output.push(array1.shift());
+            }
+            else if (array1[0] < array2[0]) {
+                output.push(array1.shift());
+            }
+            else {
+                output.push(array2.shift());
+            }
+        }
+        return output;
+    }
+}
+function merge_sort(array) {
```

Comment:
Codacy found an issue: [Identifier 'merge_sort' is not in camel case.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6336020)

Reply:
but fix this one. for the function name, the review system expects to keep the camel case notation.